README.md

```diff
@@ -1,2 +1,7 @@
-# Rent a House
-This app will allow you to rent/buy a house.
+# (about/README) Rent a House
+Key: BREAD= browse, read, edit, add, delete. This website will allow you to rent/buy a house.
+The application is still under development.
+
+The application is not yet ready as a final product. At this point the application is divided into 2 products, 
+1 is the design of the site, and the 2nd one is the code behind it.
```

Comment:
This is clearly a copy/paste error. The method you should've followed was to copy all of the information in the other README file and then delete every section that was specific to that protect and left the one that will be generic and related to this project :)